lib/persisted-model.js

```diff
@@ -550,6 +602,9 @@ module.exports = function(registry) {
     var typeName = PersistedModel.modelName;
     var options = PersistedModel.settings;
 
+    // This is just for LB 2.x
+    options.replaceOnPUT = options.replaceOnPUT === true;
```

Comment:
I believe `options.replaceOnPUT === true;` by default sets to false;

Lets see:
options.replaceOnPUT is undefined `undefined === true` => false